apps/meteor/app/livechat/server/lib/routing/AutoSelection.ts

```diff
@@ -26,6 +26,7 @@ class AutoSelection implements IRoutingMethod {
 	}
 
 	async getNextAgent(department?: string, ignoreAgentId?: string): Promise<SelectedAgent | null | undefined> {
+		// TODO: apply this extra query to other routing algorithms
```

Comment:
Can we create a task for this? To keep track of it.

Reply:
Already created: https://rocketchat.atlassian.net/browse/CTZ-42